src/ort_genai_c.h

```diff
@@ -341,6 +341,25 @@ OGA_EXPORT OgaResult* OGA_API_CALL OgaGenerator_RewindTo(OgaGenerator* generator
  */
 OGA_EXPORT OgaResult* OGA_API_CALL OgaGenerator_GetOutput(const OgaGenerator* oga_generator, const char* name, OgaTensor** out);
 
+/*
+ * \brief Returns a copy of the logits from the model as an OgaTensor on CPU. The buffer is owned by returned OgaTensor
+ *        and will be released when the OgaTensor is destroyed
+ * \param[in] generator The generator to run the GetLogits on
```

Comment:
```suggestion
 * \param[in] generator The generator get the logits from
```

'run the GetLogits' sounds like it might run the model, this should only get them if available?

Reply:
Yes, I think it might be better to raise error if logits are not ready. But since we only have GenerateNextToken() now, so we can only call GetLogits() to run logits if it's not ready and get current step's logits?